File: problems/prob09/prob9.rb
def generate_random(num)
	rand_nums = []
	(0..num).each do |x|
		rand_nums << rand(1..100)
	end
	return rand_nums.sort
end


print generate_random(10)
puts
print generate_random(20)
puts
print generate_random(5)

File: problems/prob09/prob9.js
rand_nums = []

var generate_random = function (num){
	for (var i = 0; i <= num; i++) {
		rand_nums.push(Math.floor((Math.random() * 100)));
	}
	document.write(rand_nums.sort() + "<br/>")
}

generate_random(10)
rand_nums = []
generate_random(20)
rand_nums = []
generate_random(5)